src/windows/pnpdevice.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <windows.h>
+#include <setupapi.h>
+#include <cfgmgr32.h>
+#include <initguid.h>
+#include <devguid.h>
+#include <devpkey.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
Is there a reason why you're not including all of those directly in `pnpdevice.c`?

Reply:
Originally I thought a header is needed for exposed types but I can migrate it now.